src/web_peer_connection.cpp

```diff
@@ -411,7 +411,16 @@ void web_peer_connection::write_request(peer_request const& r)
 			auto redirection = m_web->redirects.find(f.file_index);
 			if (redirection != m_web->redirects.end())
 			{
-				request += redirection->second;
+				auto const& redirect = redirection->second;
+				if (using_proxy && !redirect.empty() && (redirect[0] == '/'))
```

Comment:
but even if ``using_proxy`` is false, concatenating those two paths may result in a double slash, no?